server/middleware/populateReport.test.ts

```diff
@@ -90,8 +91,6 @@ describe('report-loading middleware', () => {
     expect(res.locals.report).toBeUndefined()
     expect(res.locals.reportUrl).toBeUndefined()
     expect(res.locals.reportSubUrlPrefix).toBeUndefined()
-    expect(next).toHaveBeenCalledWith(
-      expect.objectContaining({ message: 'populateReport() requires req.params.reportId', status: 501 }),
-    )
+    expect(next).toHaveBeenCalledWith(new NotImplemented('populateReport() requires req.params.reportId'))
```

Comment:
how come this one needs to be an instance of an error and the others are still just checking for message text and status number?

Reply:
oh, I think it's because `populateReport()` throws a `NotImplemented` error directly which is not a `SanitisedError`

Reply:
https://github.com/ministryofjustice/hmpps-incident-reporting/pull/431/commits/08ac8b828022a0eb6140b024117264f7aaa3856e